modules/op-pipelines-as-code-command-reference.adoc

```diff
@@ -115,3 +115,121 @@ The `-f` option can also accept a directory path and apply the `tkn pac resolve`
 You can override the default information gathered from the Git repository by specifying parameter values using the `-p` option. For example, you can use a Git branch as a revision and a different repository name.
 
 |===
+
+cel::
+
+:FeatureName: The `tkn pac cel` command
+include::snippets/technology-preview.adoc[]
+
+[options="header"]
+|===
+
+| Command | Description
+
+| `tkn pac cel` | Evaluate CEL (Common Expression Language) expressions against webhook payloads and headers. Because the command requires access to the raw webhook payload and header data provided by the Git provider, it is intended for administrator use for testing and debugging event filtering logic in {pac}.
+
+|===
+
+The `tkn pac cel` command uses the following syntax:
+
+[source,terminal]
+----
+tkn pac cel -b <body.json> -H <headers.txt>
+----
+
+[options="header"]
+|===
+| Option | Description
+
+| `-b, --body` | Specify the JSON payload file.
+| `-H, --headers` | Specify the headers file (plain text, JSON, or gosmee script).
+| `-p, --provider` | Specify the Git provider (`auto`, `github`, `gitlab`, `bitbucket-cloud`, `bitbucket-datacenter`, `gitea`).
+|===
+
+*Header formats*
+
+The `tkn pac cel` command supports the following header formats:
+
+.Plain HTTP header format
```

Comment:
Kindly confirm if we can use such formatted text: .Plain HTTP header format. I am not sure if this is as per CQA guidelines.